Sources/ArcGISToolkit/Components/Form/FormInputs/FormInputModel.swift

```diff
@@ -18,15 +18,18 @@ import SwiftUI
 /// A model for an input in a form.
 ///
 /// - Since: 200.3
-public class FormInputModel: ObservableObject {
+class FormInputModel: ObservableObject {
```

Comment:
I don't think this needs/should be public, should it?

Reply:
No